k8s/etcd/README.md

```diff
@@ -18,18 +17,17 @@ Popular open stacks on Kubernetes packaged by Google.
 
 ![Architecture diagram](resources/etcd-k8s-app-architecture.png)
 
-A Kubernetes StatefulSet manages all etcd Pods in this application. Each
-Pod runs a single instance of etcd. Each instance of etcd communicates each other with peer port 2380. Connection between cluster members is automatically encrypted with TLS certificates.
-TLS certificates are created and used by each instance of etcd when starting cluster service.
+A Kubernetes StatefulSet manages all of the etcd Pods in this application. Each Pod runs a single instance of etcd, and each instance of etcd communicates with the others by using peer port 2380. Connections between cluster members are automatically encrypted with TLS certificates.
+
+Each instance of etcd generates its own TLS certificate at the time that cluster service is started.
 
 Access to the etcd service is authenticated by default.
 
 # Installation
 
 ## Quick install with Google Cloud Marketplace
 
-Get up and running with a few clicks! Install this etcd app to a
-Google Kubernetes Engine cluster using Google Cloud Marketplace. Follow the
+Get up and running in just a few clicks! Install this etcd app to a Google Kubernetes Engine cluster in Google Cloud Marketplace by following these
```

Comment:
This sentence is the same across all readmes. For example: https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/k8s/jenkins#quick-install-with-google-cloud-marketplace.

Should we be consistent? Please let me know about your thought on this.